tests/garage/misc/test_wrappers.py

```diff
@@ -0,0 +1,20 @@
+import gym
+
+from garage.misc.wrappers import ResizeAndGrayscaleWrapper, StackFramesWrapper
+from tests.fixtures import TfGraphTestCase
+
+
+class TestWrappers(TfGraphTestCase):
+    def test_resize_and_gray_scale_wrapper(self):
+        env = ResizeAndGrayscaleWrapper(gym.make("Breakout-v0"), w=84, h=84)
+        assert env.observation_space.shape == (84, 84)
```

Comment:
You verified it was resized but did not verify that it's greyscale.

Reply:
can I say that the shape is now (84, 84, (1)) so the channel now becomes 1, which means grayscale?